Engine/ac/route_finder.cpp

```diff
@@ -232,12 +227,16 @@ int find_route(short srcx, short srcy, short xx, short yy, Bitmap *onscreen, int
 
   assert(num_navpoints <= MAXNAVPOINTS);
 
-//    Display("Route from %d,%d to %d,%d - %d stages", srcx,srcy,xx,yy,num_navpoints);
+#ifdef DEBUG_PATHFINDER
+  Display("Route from %d,%d to %d,%d - %d stages", srcx,srcy,xx,yy,num_navpoints);
+#endif
 
   int mlist = movlst;
   mls[mlist].numstage = num_navpoints;
   memcpy(&mls[mlist].pos[0], &navpoints[0], sizeof(int) * num_navpoints);
-//    fprintf(stderr,"stages: %d\n",num_navpoints);
+#ifdef DEBUG_PATHFINDER
+  fprintf(stderr,"stages: %d\n",num_navpoints);
```

Comment:
Also, since you are uncommenting these, please use same debug output method everywhere (preferrably Debug::Printf).